.github/workflows/ci.yaml

```diff
@@ -217,35 +200,29 @@ jobs:
       # Ignore the failure of a step and avoid terminating the job.
       continue-on-error: true
       uses: actions/cache@v3
+      id: deps_cache
       with:
         # This path is specific to Ubuntu
         path: ${{ github.workspace }}/cget
         # Look to see if there is a cache hit for the corresponding requirements file
         key: 
           ${{ matrix.os }}-cget-4-${{ hashFiles('requirements.txt', 'dev-requirements.txt') }}
+        restore-keys: |
           ${{ matrix.os }}-cget-4-
 
 
     - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install https://github.com/RadeonOpenCompute/rbuild/archive/master.tar.gz
-        rbuild prepare -d cget -s gh
-        sudo apt-get install -y lld
-    - name: Prepare timestamp
-      id: cache_timestamp
-      shell: bash 
-      run: echo timestamp="$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_OUTPUT
+      if: steps.deps_cache.outputs.cache-hit != 'true'
```

Comment:
There is no need for this. Rbuild already keeps a hash and will only rebuild if the dependencies have changed.

Reply:
I find it better to keep this than relying on rbuild.

Reply:
Why? This puts more burden on the yaml script when we already have a tool to handle this.